gemrb/core/Video/Video.cpp

```diff
@@ -133,14 +133,19 @@ void Video::SetStencilBuffer(const VideoBufferPtr& stencil)
 	stencilBuffer = stencil;
 }
 
-int Video::SwapBuffers(unsigned int fpscap)
+int Video::SwapBuffers(int fpscap)
```

Comment:
I'm not sure what the motivation is for signing this. 0 can be unlimited.

Reply:
I feed it `config.CapFPS` with `-1` (no limit), `0` (VSync) and anything else as a real cap value.

Reply:
maybe I am missing something, but this doesn't seem like it can change dynamically, and we are passing `vsync` to `CreateDriverDisplay` so it seems a bit awkward. Maybe it would be better to just set this up in `CreateDriverDisplay` and not have a parameter here at all.